### PR TITLE
add new ref in openapi.yaml & template.yaml

### PR DESCRIPTION
**Pull request by StigNorland**
Add new documentation to /resources, waiting for new endpoint to be deployed

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -80,85 +80,121 @@ paths:
...
+        - $ref: '#/components/parameters/fundingParam'
         - $ref: '#/components/parameters/fundingSourceNotParam'
+        - $ref: '#/components/parameters/fundingSourceParam'
         - $ref: '#/components/parameters/fundingSourceShouldParam'
-        - $ref: '#/components/parameters/grantIdentifierParam'
         - $ref: '#/components/parameters/grantIdentifierNotParam'
+        - $ref: '#/components/parameters/grantIdentifierParam'
         - $ref: '#/components/parameters/grantIdentifierShouldParam'
-        - $ref: '#/components/parameters/idParam'
+        - $ref: '#/components/parameters/hasFileParam'
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:119]: /hasPublicFile kommer senere? Eller alias?

File: docs/openapi.yaml [hunk 2]
```diff
@@ -552,55 +662,88 @@ paths:
...
+      name: abstract
+      in: query
+      description: "returns document with..."
+      required: false
+      schema:
+        type: string
+    abstractNotParam:
+      name: abstract_not
+      in: query
+      description: "returns document with..."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:675]: without...?

File: docs/openapi.yaml [hunk 3]
```diff
@@ -552,55 +662,88 @@ paths:
...
+      required: false
+      schema:
+        type: string
+    abstractShouldParam:
+      name: abstract_should
       in: query
       description: "Returns documents with this category code, only one category code is allowed."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:682]: Category code on abstract?

File: docs/openapi.yaml [hunk 4]
```diff
@@ -552,55 +662,88 @@ paths:
...
       required: false
       schema:
-        $ref: "#/components/schemas/CategoryEnum"
-    contributorIdParam:
-      name: contributor_id
+        type: string
+    contextTypeShouldParam:
+      name: context_type_should
+      in: query
+      description: "?"
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:704]: ?

File: docs/openapi.yaml [hunk 5]
```diff
@@ -552,55 +662,88 @@ paths:
...
-      description: "returns document with Author's name or person id(URI)."
+      description: "returns document with Author's id(URI)."
       required: false
       schema:
         type: string
     contributorNotParam:
       name: contributor_not
       in: query
-      description: "returns document without Author's names or person ids(URI), seperated by comma (+)."
+      description: "returns document without Author's ids(URI), separated by comma."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:739]: Does this one accept comma separated list? We don't specify that for other `not`'s

File: docs/openapi.yaml [hunk 6]
```diff
@@ -770,11 +913,19 @@ components:
...
+      description: "Returns documents with the grant id, also known as funding identifiers, separated by comma."
       example: "1234,4321"
       required: false
       schema:
         type: string
+    hasFileParam:
+      name: has_file
+      in: query
+      description: "Returns documents with visibleForNonOwner=true/false"
+      example: "hasFile=true ,hasFile=false"
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:925]: Wrong comma placement

File: docs/openapi.yaml [hunk 7]
```diff
@@ -785,14 +936,35 @@ components:
...
+      name: instance_type_not
+      in: query
+      description: "returns document without this instanceType."
+      required: false
+      schema:
+        type: string
+    instanceTypeShouldParam:
+      name: instance_type_should
+      in: query
+      description: "returns document with these instanceTypes"
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:967]: ... separated by comma

File: docs/openapi.yaml [hunk 8]
```diff
@@ -1024,34 +1216,69 @@ components:
       schema:
         type: string
         format: date
-    queryParam:
-      name: query
+    publisherIdNotParam:
+      name: publisher_id_not
       in: query
-      description: "free text search, all words are searched for in all fields, only documents containing all words are returned"
+      description: "returns document without this Id."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:1222]: ... publisher Id."

File: docs/openapi.yaml [hunk 9]
```diff
@@ -1024,34 +1216,69 @@ components:
...
       in: query
-      description: "Sorts on the given field. Default sort order is on 'published_date' in ascending order."
+      description: "returns document with this Id"
+      required: false
+      schema:
+        type: string
+    publisherIdShouldParam:
+      name: publisher_id_should
+      in: query
+      description: "returns document with these Ids."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:1236]: ...these publisher Ids, separated by comma."

File: docs/openapi.yaml [hunk 10]
```diff
@@ -1024,34 +1216,69 @@ components:
...
+      name: publisher_not
+      in: query
+      description: "returns document without this publisher."
+      required: false
+      schema:
+        type: string
+    publisherParam:
+      name: publisher
+      in: query
+      description: "returns document with this publisher."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:1250]: URI?

File: docs/openapi.yaml [hunk 11]
```diff
@@ -1024,34 +1216,69 @@ components:
...
+      required: false
+      schema:
+        type: string
+    publisherShouldParam:
+      name: publisher_should
+      in: query
+      description: "returns document with these publishers."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:1257]: ...., separated by comma"

File: docs/openapi.yaml [hunk 12]
```diff
@@ -1164,6 +1419,124 @@ components:
       required: false
       schema:
         type: string
+    userAffiliationParam:
+      name: user_affiliation
+      in: query
+      description: "returns document with..."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:1425]: with....?

File: docs/openapi.yaml [hunk 13]
```diff
@@ -1164,6 +1419,124 @@ components:
...
+      required: false
+      schema:
+        type: string
+    userAffiliationNotParam:
+      name: user_affiliation_not
+      in: query
+      description: "returns document with..."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:1432]: with....?

File: docs/openapi.yaml [hunk 14]
```diff
@@ -1164,6 +1419,124 @@ components:
...
+      required: false
+      schema:
+        type: string
+    userAffiliationShouldParam:
+      name: user_affiliation_should
+      in: query
+      description: "returns document with..."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:1439]: with....?

File: docs/openapi.yaml [hunk 15]
```diff
@@ -1164,6 +1419,124 @@ components:
...
+      required: false
+      schema:
+        type: string
+    #--- aliases - maybe delete?
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:1443]: Should grantId be here as well?

**Review by joachimjorgensen — APPROVED** [review]
Approved with a few comments :)

File: docs/openapi.yaml (hunk 1 above)
Comment (StigNorland) [reply docs/openapi.yaml:119]: JA 👍 (vi er ikke ferdige med hasFile/hasPublicFile)

**Pull request closed without merging**